### DISCUSSION TITLE
Cannot accept autocomplete suggestions to prompt

### DISCUSSION THREAD

**Discussion by edmondop** [CopilotC-Nvim/CopilotChat.nvim#388] [Q&A]
When I am writing a prompt within the Copilot Chat, sometimes I see very useful suggestions to complete the prompt and this would save me quite a bit amount of time. However, traditional hotkeys that I can use in my buffer won't work, so I need always to type the prompt myself.

How should I troubleshoot the issue? `:checkhealth CopilotChat` doesn't show any problems. Both Tab and the right arrow are not working

**Comment by sjonpaulbrown-cc** [comment]
@edmondop While working on MacOs, I experienced this same issue. By default, the key for completions is set to `<Tab>`. This did not work for me, but the following change to the mapping enabled me to properly use my Tab key for auto completion in insert mode. I believe it is something to do with the key mapping on the OS, but after I found a solution, I didn't dig any further. If `<C-t>` does not work for you, I would look into what the Tab key is mapped to in your nvim setup.

```
        mappings = {
            complete = {
                insert = '<C-t>'
            },
        },
```
